src/api/types/test-helpers/index.ts

```diff
@@ -1,11 +1,13 @@
 import Context from '@keystone-6/core/types';
+/* eslint-disable */
```

Comment:
Can you please add a comment why we are done this and a `TODO` on a long term fix on the ticket please.

Reply:
done - no TODO as not sure if it will ever be fixed